client/src/plugins/camunda-plugin/start-instance-tool/StartInstanceTool.js

```diff
@@ -267,6 +267,8 @@ export default class StartInstanceTool extends PureComponent {
         endpoint
       } = deploymentConfig;
 
+      startConfiguration.businessKey = startConfiguration.businessKey === '' ? null : startConfiguration.businessKey;
```

Comment:
Let's do some idiomatic JS.

```suggestion
      startConfiguration.businessKey = startConfiguration.businessKey || null;
```

Reply:
Will check on how we can test this.